app/helpers/sufia_helper.rb

```diff
@@ -5,25 +5,21 @@ module SufiaHelper
 
   # override SufiaHelperBehavior method for institutional_access? condition
   def sufia_thumbnail_tag(document, options)
-    # collection
     if document.collection?
-      content_tag(:span, "", class: "glyphicon glyphicon-th collection-icon-search")
```

Comment:
Out of curiosity, are double quotes only used in case of string interpolation in Ruby, and single quotes are always preferred?